### PR TITLE
Bug 1789992 - Add new proxy attributes to metadata

### PR DESCRIPTION
**Pull request by relud**
~requires~ blocks https://github.com/mozilla/gcp-ingestion/pull/2271

### CODE REVIEW

**Comment by relud** [comment]
cc @whd

**Comment by whd** [comment]
I imagine https://github.com/mozilla/gcp-ingestion/pull/2271 requires this and not the other way around i.e. we'll want the schemas deployed before trying to populate those fields.

**Comment by relud** [comment]
> I imagine [mozilla/gcp-ingestion#2271](https://github.com/mozilla/gcp-ingestion/pull/2271) requires this and not the other way around i.e. we'll want the schemas deployed before trying to populate those fields.

yes, the new fields being in metadata means that they will not be recorded until these schema changes are deployed.

File: templates/include/metadata/ingestionCommonMetadata.1.schema.json [hunk 1]
```diff
@@ -93,4 +93,18 @@
       "type": "string"
     }
   }
+},
+"proxy": {
+  "type": "object",
+  "description": "Results of Proxy detection based on X-Forwarded-For",
+  "properties": {
+    "list_versions": {
+      "description": "The specific proxy subnet list versions used for this detection",
```
Comment (whd) [inline templates/include/metadata/ingestionCommonMetadata.1.schema.json:102]: CSV?

**Review by whd — APPROVED** [review]
LGTM. I'd like to be responsible for merging/deploying this when I'm working on https://mozilla-hub.atlassian.net/browse/DSRE-1195

File: templates/include/metadata/ingestionCommonMetadata.1.schema.json (hunk 1 above)
Comment (relud) [reply templates/include/metadata/ingestionCommonMetadata.1.schema.json:102]: right now it's semicolon separated, <source>:<timestamp>, but I'm not attached to that format, it was just the first thing that came to mind when implementing:
https://github.com/mozilla/gcp-ingestion/blob/7780f9e073a16bbf9d43cf47bcda33a6e2585551/ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParseProxyTest.java#L77-L79

**Pull request closed without merging**